src/components/partials/IdentityList.vue

```diff
@@ -1,12 +1,19 @@
 <template>
     <div class="pb-5 desktop-header">
         <p class="h4">
-            {{ searchResults.length }} Search Results for "{{ lastSearchTerm }}"
-            in "{{ chainName }}"
+            {{ lastTotalItemCount }} Search
+            <span v-if="lastTotalItemCount > 1">Results</span>
+            <span v-if="lastTotalItemCount === 1">Result</span> for "{{
+                lastSearchTerm
+            }}" in "{{ chainName }}"
```

Comment:
@sosan-neikbeen formatting wrong and use v-else please

```html
<p class="h4">
    {{ lastTotalItemCount }} Search
    <span v-if="lastTotalItemCount === 1">Result</span>
    <span v-else>Result</span> 
    for "{{ lastSearchTerm }}" in "{{ chainName }}"
</p>
```

Reply:
We don't need this check. because we are going to redirect to identity view if we got one result so I am going to remove it completely.